File: backend-dotnet/SahayaSetu/DTO/NGO/DonationResponseDto.cs
namespace SahayaSetu.DTO.NGO
{
    public class DonationResponseDto
    {
        public int DonationId { get; set; }
        public string? DonorName { get; set; }
        public string? DonorEmail { get; set; }
        public decimal Amount { get; set; }
        public string? FundraiserTitle { get; set; }
        public DateTime DonationDate { get; set; }
        public bool ResponseStatus { get; set; }
        public string? ResponseMessage { get; set; }
    }
}

File: backend-java/sahayaSetu/src/main/java/com/sahayaSetu/dtos/DonationResponseDto.java
package com.sahayaSetu.dtos;

import java.math.BigDecimal;
import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DonationResponseDto {

    private Long donationId;
    private String donorName;
    private String donorEmail;
    private BigDecimal amount;
    private String fundraiserTitle;
    private LocalDateTime donationDate;
    private Boolean responseStatus;
    private String responseMessage;
}
